packages/daemon/src/utils/helpers.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+ import { StringMap } from '../types';
+
+export const stringMapIterator = (stringMap: StringMap<unknown>): [string, unknown][] => (Object.entries(stringMap));
```

Comment:
```suggestion
export function stringMapIterator<T>(stringMap: StringMap<T>): [string, T][] => (Object.entries(stringMap));
```

Reply:
Thanks, done in 2f912541f296b7c34ce3555b76340517444338f6